sql/event_data_objects.cc

```diff
@@ -1249,6 +1252,8 @@ Event_timed::get_create_event(THD *thd, String *buf)
 
   if (status == Event_parse_data::ENABLED)
     buf->append(STRING_WITH_LEN("ENABLE"));
+  else if (status == Event_parse_data::SLAVESIDE_ENABLED)
+    buf->append(STRING_WITH_LEN("ENABLE"));
```

Comment:
Shouldn't this be "ENABLE ON SLAVE"?

Reply:
You are right! I missed this one. Thanks.